### DISCUSSION TITLE
Logs stopped

### DISCUSSION THREAD

**Discussion by yula-21** [Security-Onion-Solutions/securityonion#5801] [Unsupported Versions] [answered]
Hi,
Im trying to use zeek with the tool bropy to create a baseline for the network.

As far a I know the tool uses the conn logs from /nsm/zeek/logs to create the network baseline and its not supposed to ruin any of the configuration files.

On first Installation I've seen logs flowing but it stopped quickly after. 
I cant see any logs in the kibana discover page (other than the very first day of installation).

Ive noticed the following error in the logstash logs:

**_[WARN] [logstash.inputs.redis] Redis connection problem {:exception=>#<Redis::CannotConnectError: Error connecting to Redis on nsm:9696 (Redis::TimeoutError)>}_**

So it looks like theres a problem forwarding the logs, does anyone have any idea why it happens or how to fix it?

**Comment by dougburks** [comment, marked as answer]
bropy is several years old and appears to be unmaintained. It was originally designed for Bro (not Zeek) on an older version of Security Onion that is now End Of Life. You might be able to make bropy work, but we can't provide any support for it.
